### ISSUE TITLE
does a zed mini camera need Nvidia GPU to work

### ISSUE THREAD

**Issue by svabhishek29** [stereolabs/zed-python-api#110]
Hi, 

i was looking at purchasing a new zed mini camera. on the system req part the page says minimum a nvidia GTX 1070 is needed for MR to run at 90FPS.

i just wanted to know if a CUDA GPU is needed to run the camera and get depth and pose data? 

if not then why is the 1070 card a minimum req for Mixed reality? is it to be able to run the games at proper frames rates?

Thanks

**Comment by obraun-sl** [comment]
Hi,
A NVIDIA GPU is required to run the ZED SDK (get depth and pose data and more...) . However you can use the camera as a standard USB camera (UVC) but you won't have the depth and pose, computed by the ZED SDK then.

For Mixed reality, the camera is often used with a HMD (Oculus, HTC Vive....) and to have a proper framerate compatible with those device, a high-end graphic card is needed. Note that this is about the same requirement than using an Oculus or a Vive, i.e. GTX1060/1070/1080....
Basically, if you have a graphic card compatible with Oculus or Vive, you will be able to run the ZED SDK for mixed reality at a good fps.
